docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -214,13 +214,20 @@ To support polymorphic deserialization, create a converter like the example in [
 
 ### Deserialization of object properties
 
-When `Newtonsoft.Json` deserializes to `object` properties in POCOs or in dictionaries of type `Dictionary<string, object>`, it:
+When `Newtonsoft.Json` deserializes to <xref:System.Object>, it:
 
 * Infers the type of primitive values in the JSON payload (other than `null`) and returns the stored `string`, `long`, `double`, `boolean`, or `DateTime` as a boxed object. *Primitive values* are single JSON values such as a JSON number, string, `true`, `false`, or `null`.
 * Returns a `JObject` or `JArray` for complex values in the JSON payload. *Complex values* are collections of JSON key-value pairs within braces (`{}`) or lists of values within brackets (`[]`). The properties and values within the braces or brackets can have additional properties or values.
 * Returns a null reference when the payload has the `null` JSON literal.
 
-<xref:System.Text.Json> stores a boxed `JsonElement` for both primitive and complex values within the `System.Object` property or dictionary value. However, it treats `null` the same as `Newtonsoft.Json` and returns a null reference when the payload has the `null` JSON literal in it.
+<xref:System.Text.Json> stores a boxed `JsonElement` for both primitive and complex values whenever deserializing to <xref:System.Object>, for example:
+
+* An `object` property
```

Comment:
I think these should have `.` at the end.

Reply:
See https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28751

>If all list elements are short phrases (three words or fewer), don’t end them with periods, even if they form a complete sentence together with the list introduction. 

This is an edge case, as 2 bullets are 3 words and 2 are 4 words. But strictly speaking it doesn't match the rule.